src/content/docs/cypher/syntax.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Syntax
+description: Basics of the Cypher query language syntax
+---
+
+In this page, we list the syntactic features of Cypher as implemented in Kùzu. As described in the
+[overview](/cypher) page, Cypher is a declarative graph query language, and Kùzu's implementation is
+based on [openCypher](https://opencypher.org/resources/).
+
+## Parsing
```

Comment:
Parsing sounds a bit too abstract for this section.

I would split them into 
```
Encoding (discuss ascii, unicode, ...)

Escaping (discuss special character, using space as a table name, etc)

Multiline statement (discuss line breaking and ; termination)
```
But this is up to you.

Reply:
I used the same subheader that Neo4j did in the start of their Cypher manual, under syntax. But ours is less verbose and all in one page (Neo4j breaks theirs down into multiple pages).

Reply:
Ok, fixed this by adding more subheadings to the "Parsing section".